utils/ioprovenance.py

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env python
+"""
+Utility funcitons for provenance of feature files.
+
+Authors: Phillip J. Wolfram
+Last Modified: 09/29/2016
+"""
+
+import os
+import sys
+import socket
+import subprocess
+import numpy as np
+import datetime
+
+def provenance_command(): #{{{
+    cwd = os.getcwd()
```

Comment:
I don't think we need a scrubber.  People can use their best judgment about what to make public, but I now see how much of this stuff is in all the NetCDF files we have around so I guess that ship has definitely sailed.